TTHAnalysis/python/plotter/functions.cc

```diff
@@ -191,8 +191,347 @@ float DPhi_CMLep_Zboost(float l_pt, float l_eta, float l_phi, float l_M, float l
   return deltaPhi(l1.Phi(),Z.Phi());
 }
 
+// May 2016: This is func which takes the 4vectors of MET, L1,L2 and reconstructs 2 taus and returns the effective Z(-->tautau) mass.                                                                                       
```

Comment:
Speaking with @nachosandres actually the situation seems to be that this stuff had been removed from functions.cc but not put anywhere else, so its inclusion back has been an emergency patch to have stuff ready for the FSR!

(still suggesting to leave it untouched until #60 is merged, as agreed on skype)